configs/settings-example.yml

```diff
@@ -0,0 +1,55 @@
+# Variables for ansible provisioning
+#  https://docs.ansible.com/ansible/latest/playbooks_variables.html
+
+env: "dev" # dev | production - to enable TLS change mediawiki.protocal to 'https' below
+#apt_proxy: IP address # Install apt-cacher-ng on the host to avoid downloading packages again.
+
+mediawiki:
+  domain: hitchwiki.test
+  protocol: http # change to 'https' to enable TLS
+  debug: False
+  cache: False
+  branch: REL1_29
+  sitename: Hitchwiki
+  metanamespace: Hitchwiki
+  cookiedomain: .hitchwiki.test
+  db_prefix: "hw_"
+  protocol: http # change to https with enabled TLS
+  owner: "{{ apache.user }}:{{ apache.group }}"
+  db_prefix: "hw_"
+  visible_to_search_engines: True
+  user:
+    Hitchwiki: "{{ db.phpmyadmin_password }}"
+    Hitchbot: "{{ db.phpmyadmin_password }}"
+    Hitchhiker: "{{ db.phpmyadmin_password }}"
+  smtp: # MailDev is running at port 1025 at dev server https://github.com/djfarrelly/MailDev
```

Comment:
SMTP settings could also be at top level since they're gonna be the same for anything else than just MediaWiki.

Reply:
done